### PR TITLE
bundledata: add the expose field to the service spec.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by urosj** [comment]
:+1:

**Comment by frankban** [comment]
Thank you Uros for the review!
:shipit:

File: bundledata.go [hunk 1]
```diff
@@ -121,6 +121,9 @@ type ServiceSpec struct {
 	//     wordpress wordpress lxc:0 kvm:new
 	To []string `bson:",omitempty" json:",omitempty" yaml:",omitempty"`
 
+	// Expose reports whether the service must be exposed.
```
Comment (rogpeppe) [inline bundledata.go]: s/reports/holds/

(I think - this isn't a function but a field)

**Comment by rogpeppe** [comment]
LGTM with one trivial thought.

File: bundledata.go (hunk 1 above)
Comment (frankban) [inline bundledata.go]: Done.

**Comment by frankban** [comment]
$$merge$$

**Pull request merged**